src/lib/inputleap/ServerApp.h

```diff
@@ -118,11 +118,9 @@ class ServerApp : public App {
 
 // configuration file name
 #if SYSAPI_WIN32
-#define USR_CONFIG_NAME "input-leap.sgc"
-#define SYS_CONFIG_NAME "input-leap.sgc"
+#define CONFIG_NAME "input-leap.sgc"
```

Comment:
This should be split into a separate PR. Currently the PR is for 'not using hidden files for config on Linux' - but this change affects win32.

Reply:
OK, thanks for the clarification.